lib/go-rfc/cachecontrol_test.go

```diff
@@ -0,0 +1,44 @@
+package rfc
+
+import (
+	"testing"
+	"time"
+)
+
+/*
```

Comment:
Nitpick: the license header should go immediately after the `package`, but before the `import`.